app/charts/map/helpers.spec.ts

```diff
@@ -1,45 +1,108 @@
-import { Bounds } from "@/charts/shared/use-width";
+import { renderHook } from "@testing-library/react-hooks";
+
 import { BBox } from "@/configurator/config-types";
 
-import { BASE_VIEW_STATE, initializeViewState } from "./helpers";
-
-describe("initialization of view state", () => {
-  const bbox = [
-    [0, 0],
-    [0, 0],
-  ] as BBox;
-  const chartDimensions = {
-    width: 400,
-    height: 400,
-  } as Bounds;
-
-  it("should use base view state when bbox is undefined", () => {
-    const viewState = initializeViewState({
-      chartDimensions,
-      bbox: undefined,
-      locked: true,
+import {
+  BASE_VIEW_STATE,
+  useViewState,
+  ViewStateInitializationProps,
+} from "./helpers";
+
+const width = 400;
+const height = 200;
+const lockedBBox = [
+  [6.8965001, 46.3947983],
+  [6.8965001, 46.3947983],
+] as BBox;
+const featuresBBox = [
+  [6.4965001, 46.2947983],
+  [6.9965001, 46.4947983],
+] as BBox;
+
+describe("useViewState", () => {
+  it("should properly set defaultViewState", () => {
+    const { result, rerender } = renderHook<
+      ViewStateInitializationProps,
+      ReturnType<typeof useViewState>
+    >((props: ViewStateInitializationProps) => useViewState(props), {
+      initialProps: {
+        width,
+        height,
+        lockedBBox: undefined,
+        featuresBBox: undefined,
+      },
     });
 
-    expect(viewState).toEqual(BASE_VIEW_STATE);
+    // If featuresBBox was not privded, defaultViewState should equal BASE_VIEW_STATE.
+    expect(result.current.defaultViewState.latitude).toEqual(
+      BASE_VIEW_STATE.latitude
+    );
+    expect(result.current.defaultViewState.longitude).toEqual(
+      BASE_VIEW_STATE.longitude
+    );
+    expect(result.current.defaultViewState.zoom).toEqual(BASE_VIEW_STATE.zoom);
```

Comment:
Nit: I think the 3 expect calls could be done via one toEqual call (expect(defaultVIewState).toEqual(BASE_VIEW_STATE)